app/src/interfaces/useBioHubApi-interfaces.ts

```diff
@@ -102,3 +102,23 @@ export interface IMedia {
   file_name: string;
   encoded_file: string;
 }
+
+/**
+ * Upload project artifacts request object.
+ * @interface IUploadProjectArtifactRequest
+ */
+export interface IUploadProjectArtifactsRequest {
+  artifacts: {
+    file_name: string;
+    encoded_file: string;
+  }[];
+}
+
+/**
+ * Upload project artifacts response object.
+ */
+export interface IUploadProjectArtifactsResponse {
+  success: boolean;
+  error_code?: number;
+  error_message?: string;
+}
```

Comment:
This is listed as the response to the `uploadProjectArtifacts function`, but the actual response is some kind of array of s3 objects (with whatever content they have).

Reply:
adjusted this to return the Key for each uploaded file